docs/usage/config-presets.md

```diff
@@ -96,7 +96,7 @@ Here is how you would use these in your Renovate config:
 
 ```json
   "extends": [
-    ":labels(depedendencies,devops)",
+    ":labels(dependencies,devops)",
```

Comment:
Revert

Reply:
Sure. Out of curiosity, is this a compatibility thing?

Reply:
(There are only 686 results for this token according to Google vs 169 million for the other. GitHub has 1495 code results vs 72 million.)